src/thor/timedistancebssmatrix.cc

```diff
@@ -590,13 +590,13 @@ void TimeDistanceBSSMatrix::SetDestinationsOneToMany(
   // For each destination
   uint32_t idx = 0;
 
-  // Only skip outbound edges if we have other options
-  bool has_other_edges = false;
-
   for (const auto& loc : locations) {
     // Set up the destination - consider each possible location edge.
     bool added = false;
 
+    // Only skip outbound edges if we have other options
+    bool has_other_edges = false;
```

Comment:
I see, it's per individual location not for all locations. Great catch!

Reply:
copy-paste is evil :D